internal/sabnzbd/collector/collector.go

```diff
@@ -189,7 +189,7 @@ func NewSabnzbdCollector(config *config.SabnzbdConfig) (*SabnzbdCollector, error
 func (s *SabnzbdCollector) doRequest(mode string, target interface{}) error {
 	params := client.QueryParams{}
 	params.Add("mode", mode)
-	return s.client.DoRequest("/sabnzbd/api", target, params)
+	return s.client.DoRequest("/api", target, params)
```

Comment:
While I understand the thought process here, this would be a breaking change for existing users :-/ Is this actively breaking your setup? If not, we can create an issue and make this change when we do a major version bump. If it is breaking your setup, we probably want to put this behind a flag for now.

Reply:
This is actively breaking my setup. Can we introduce a defaulted environment variable here?

Reply:
@rtrox I am ok with this breaking change going into a minor release.